Core/RadioHandler.cpp

```diff
@@ -203,8 +226,8 @@ bool RadioHandlerClass::Stop()
 		submit_thread.join();
 		DbgPrintf("submit_thread join1\n");
 
-		hardware->FX3producerOff();     //FX3 stop the producer
 	}
+	hardware->FX3producerOff();     //FX3 stop the producer
```

Comment:
it will get called in Stop. We should either remove the other place to call this or delete this line.

Reply:
Where is the other place?